### ISSUE TITLE
Custom scrollbar

### ISSUE THREAD

**Issue by KarishmaVanwari** [Team-Vaayushastra/Team-Vaayushastra.github.io#21]
Adding a custom scrollbar would improve the look of the website.

**Comment by KarishmaVanwari** [comment]
Hey, can I work on this for Hacktoberfest 2022? You can let me know your colour preferences, if any.

**Comment by aiden506** [comment]
Yes match the scrollbar with the color blue or the color of the navbar
